Makefile

```diff
@@ -35,7 +35,7 @@ generate:
 
 .PHONY: audit
 audit:
-	go list -m all | nancy sleuth
+	go list -m all | nancy sleuth --exclude-vulnerability-file ./.nancy-ignore
```

Comment:
I dont think you need the flag - nancy sleuth by default should ignore vulnerabilities in .nancy-ignore file

Reply:
removed